src/fixate/drivers/ftdi.py

```diff
@@ -390,7 +388,7 @@ def configure_bit_bang(
         :param clk_mask: CBUS Pin for clock. 2 Default for Relay Matrix
         :param data_mask: CBUS Pin for data. 4 Default for Relay Matrix
         :param invert_mask: Mask for inverting. 0b111 For all inverted 0b000 for all non inverted
-        based on MSB 0b<latch><clock><data> LSB
+            based on MSB 0b<latch><clock><data> LSB
```

Comment:
similar here? although I don't even understand the original

```
:param invert_mask: Mask for inverting. 

* 0b111 For all inverted
* 0b000 for all non inverted
* based on MSB 0b<latch><clock><data> LSB
```

and worthwhile moving to the new markup for readability?

```
Params:
    clk_mask: CBUS Pin for clock. 2 Default for Relay Matrix
    data_mask: CBUS Pin for data. 4 Default for Relay Matrix
    invert_mask: Mask for inverting. 

     * 0b111 For all inverted
     * 0b000 for all non inverted
     * based on MSB 0b<latch><clock><data> LSB
```

Reply:
I'm not keen on reformatting stuff at the moment. Primary goal was to ensure the new switching stuff was included and them did a few drive by fix-ups.

I've ended up re-wording this param description anyway.